action.yml

```diff
@@ -20,7 +20,11 @@ inputs:
     description: 'the number of cores to use for the emulator'
     default: 2
   ram-size:
-    description: 'size of RAM to use for this AVD, in KB or MB, denoted with K or M. - e.g. `2048M`'
+    description: 'size of RAM to use for this AVD, in MB, - e.g. `2048`'
+    default: 2048
```

Comment:
any reason we need to add a `default` here?

Reply:
I've set as default the value that Android Studio emulator uses, but I think it's not really necessary. The same answer about heap size.